src/lib/apify_command.js

```diff
@@ -2,18 +2,31 @@ const { Command } = require('@oclif/command');
 const { finished } = require('stream');
 const { promisify } = require('util');
 const { argsToCamelCase } = require('./utils');
+const { mixpanel, getOrCreateLocalDistinctId, isTelemetryEnabled } = require('./telemetry');
+const { detectInstallationType } = require('./version_check');
 
 /**
  * Adding parsing flags to oclif Command class
  */
 class ApifyCommand extends Command {
     parse(cmd) {
```

Comment:
I would not add the tracking to the `parse` method, but instead override the `finally` method on this class, and track only after the command finishes. That way you can easily track the errors that happen in the command.

And you could add some `trackingDetails` property on this class, which you could fill in the `run` method e.g. with the actor template properties, and add it to the tracked data passed to `mixpanel.track()`. That way you can track all the properties of one command invocation in one tracking call.

Reply:
If could also go to hooks - `post run` hook would work well.

Reply:
`postrun` hook is not executed if command finishes with an error unfortunately:
https://oclif.io/docs/hooks#lifecycle-events